manuscript/main.tex

```diff
@@ -255,19 +255,16 @@ \subsubsection*{Cohort overview and summary data}
 age-appropriate cognitive and behavioral assessments
 using full-scale composite scores from the MSEL, WPPSI-4 and WISC-V,
 and neuroimaging measures of cortical and sub-cortical morphology. 
-To measure cognitive function, we used
-age-appropriate assessments that can be 
 Different assessment instruments were used depending on the child's age
 due to the unique age-range and psychometric properties measured by each
 (i.e., the MSEL for children under 4 years of age, The WPPSI-3 for 3-6 year-olds,
-and the WISC-V for children older than 6), but were normalized to a common scale (Figure 1B), 
-
+and the WISC-V for children older than 6), but were normalized to a common scale (Figure 1B),
 As expected, the greatest differences in microbial taxa were observed across
-child age (Figure 1B-C, PCoA axis 1), while older children were primarily
+child age (Figure 1C-D, PCoA axis 1), while older children were primarily
 stratified into Bacteroidetes-dominant, Firmicutes-dominant, or
 high-\emph{Prevotella copri} (Supplementary Figure 2). Overall variation
 in gut microbial genes and also in brain volume profiles was similarly
```

Comment:
@vanjakle had 

> Overall, subject-to-subject variation in gut microbial genes and brain volume profiles were similarly driven largely by subject age

So, comma after "Overall" and "were driven". But I think we're talking about "driving overall variation", rather than "overall, bacteria drive variation".